Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/mongodb/swift-bson", .upToNextMajor(from: "2.0.0")),
         .package(url: "https://github.com/mongodb/swift-mongoc", .upToNextMajor(from: "2.0.0")),
-        .package(url: "https://github.com/Quick/Nimble.git", .upToNextMajor(from: "8.0.0"))
+        .package(url: "https://github.com/Quick/Nimble.git", .exact("8.0.2"))
```

Comment:
Nimble 8.0.3+ has some issues with the code coverage tool. It seemed easiest to just pin to this version.